[0.000000] (-) TimerEvent: {}
[0.001021] (imu_publisher) JobQueued: {'identifier': 'imu_publisher', 'dependencies': OrderedDict()}
[0.001292] (joy_subscriber) JobQueued: {'identifier': 'joy_subscriber', 'dependencies': OrderedDict()}
[0.001661] (imu_publisher) JobStarted: {'identifier': 'imu_publisher'}
[0.025801] (joy_subscriber) JobStarted: {'identifier': 'joy_subscriber'}
[0.099399] (-) TimerEvent: {}
[0.200109] (-) TimerEvent: {}
[0.300765] (-) TimerEvent: {}
[0.402273] (-) TimerEvent: {}
[0.506071] (-) TimerEvent: {}
[0.606687] (-) TimerEvent: {}
[0.707175] (-) TimerEvent: {}
[0.808072] (-) TimerEvent: {}
[0.909061] (-) TimerEvent: {}
[1.009682] (-) TimerEvent: {}
[1.110219] (-) TimerEvent: {}
[1.210713] (-) TimerEvent: {}
[1.311157] (-) TimerEvent: {}
[1.394834] (imu_publisher) StderrLine: {'line': b'Traceback (most recent call last):\n  File "/usr/lib/python3/dist-packages/colcon_parallel_executor/executor/parallel.py", line 67, in execute\n    loop.run_until_complete(future)\n  File "/usr/lib/python3.8/asyncio/base_events.py", line 603, in run_until_complete\n    self.run_forever()\n  File "/usr/lib/python3.8/asyncio/base_events.py", line 570, in run_forever\n    self._run_once()\n  File "/usr/lib/python3.8/asyncio/base_events.py", line 1823, in _run_once\n    event_list = self._selector.select(timeout)\n  File "/usr/lib/python3.8/selectors.py", line 468, in select\n    fd_event_list = self._selector.poll(timeout, max_ev)\nKeyboardInterrupt\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/usr/lib/python3/dist-packages/colcon_core/executor/__init__.py", line 91, in __call__\n    rc = await self.task(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/__init__.py", line 93, in __call__\n    return await task_method(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_ros/task/ament_python/build.py", line 105, in build\n    return await extension.build(additional_hooks=additional_hooks)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/build.py", line 81, in build\n    available_commands = await self._get_available_commands(\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/build.py", line 171, in _get_available_commands\n    output = await check_output(\n  File "/usr/lib/python3/dist-packages/colcon_core/subprocess.py", line 128, in check_output\n    assert not rc, f\'Expected {args} to pass: {stderr_data}\'\nAssertionError: Expected [\'/usr/bin/python3\', \'setup.py\', \'--help-commands\'] to pass: \n'}
[1.395912] (imu_publisher) JobEnded: {'identifier': 'imu_publisher', 'rc': 'SIGINT'}
[1.411266] (-) TimerEvent: {}
[1.467874] (joy_subscriber) JobEnded: {'identifier': 'joy_subscriber', 'rc': 'SIGINT'}
[1.479082] (-) EventReactorShutdown: {}
